ps_searchbar.tpl

```diff
@@ -23,12 +23,11 @@
 *  International Registered Trademark & Property of PrestaShop SA
 *}
 
-<div id="search_widget" data-search-controller-url="{$search_controller_url}">
-	<form method="get" action="{$search_controller_url}">
-		<input type="hidden" name="controller" value="search">
-		<input type="text" name="s" value="{$search_string}">
-		<button type="submit">
-			{l s='Search' d='Modules.Searchbar.Shop'}
-		</button>
-	</form>
+<div id="search_widget" class="search-widgets" data-search-controller-url="{$search_controller_url}">
+  <form method="get" action="{$search_controller_url}">
+    <input type="hidden" name="controller" value="search">
+    <i class="material-icons search">&#xE8B6;</i>
+    <input type="text" name="s" value="{$search_string}" placeholder="{l s='Search our catalog' d='Modules.Searchbar.Catalog'}" aria-label="{l s='Search' d='Shop.Theme.Catalog'}">
```

Comment:
```
Modules.Searchbar.Catalog
```

I don't think this domain is right, wdyt @LouiseBonnard ?

Reply:
```suggestion
    <input type="text" name="s" value="{$search_string}" placeholder="{l s='Search our catalog' d='Shop.Theme.Catalog'}" aria-label="{l s='Search' d='Shop.Theme.Catalog'}">
```